### PR TITLE
Add changes to start code

### PR DESCRIPTION
**Pull request by MaxJGruber**

### CODE REVIEW

**Comment by MaxJGruber** [comment]
Hey @finnwoelm-sdsn!

To the best of my knowledge and the changes you made to the global dashboard, the starter pack should be squeaky clean and ready!

Let me know if there is anything you think should be modified.

File: next.config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (finnwoelm-sdsn) [inline next.config.js:1]: delete?

Comment (finnwoelm-sdsn) [reply next.config.js:1]: Otherwise, everything looks great 😊 well done!!

**Comment by finnwoelm-sdsn** [comment]
Fixes #5

**Pull request merged**